status_test.go

```diff
@@ -61,3 +61,30 @@ func TestStatusList(t *testing.T) {
 		t.Fatal("Incorrect entry path: ", entry.IndexToWorkdir.NewFile.Path)
 	}
 }
+
+func TestStatusNothing(t *testing.T) {
+	t.Parallel()
+	repo := createTestRepo(t)
+	defer cleanupTestRepo(t, repo)
+
+	seedTestRepo(t, repo)
+
+	opts := &StatusOptions{}
+	opts.Show = StatusShowIndexAndWorkdir
+	opts.Flags = StatusOptIncludeUntracked | StatusOptRenamesHeadToIndex | StatusOptSortCaseSensitively
```

Comment:
nit:

```suggestion
	opts := &StatusOptions{
		Show:  StatusShowIndexAndWorkdir,
		Flags: StatusOptIncludeUntracked | StatusOptRenamesHeadToIndex | StatusOptSortCaseSensitively,
	}
```

this is a bit [more idiomatic](https://golang.org/doc/effective_go.html#composite_literals).

Reply:
This was probably me looking for a `DefaultStatusOptions` and not finding it then not going back and fixing it